test/basic-querying.test.js

```diff
@@ -375,39 +383,40 @@ describe('basic-querying', function() {
       });
     });
 
-    it('should support number "gt" that is satisfied by null value', function(done) {
-      User.find({order: 'seq', where: {order: {'gt': null},
-      }}, function(err, users) {
-        should.not.exist(err);
-        users.should.have.property('length', 0);
-        done();
+    bdd.itIf(!isCassandraConnector, 'should support number "gt" that is satisfied by null value',
+      function(done) {
+        User.find({order: 'seq', where: {order: {'gt': null}}}, function(err, users) {
+          should.not.exist(err);
+          users.should.have.property('length', 0);
+          done();
+        });
       });
-    });
 
-    it('should support number "lt" that is not satisfied by null value', function(done) {
-      User.find({order: 'seq', where: {order: {'lt': null},
-      }}, function(err, users) {
-        should.not.exist(err);
-        users.should.have.property('length', 0);
-        done();
+    bdd.itIf(!isCassandraConnector, 'should support number "lt" that is not satisfied by null value',
+      function(done) {
+        User.find({order: 'seq', where: {order: {'lt': null}}}, function(err, users) {
+          should.not.exist(err);
+          users.should.have.property('length', 0);
+          done();
+        });
       });
-    });
 
-    it('should support string "gte" that is satisfied by null value', function(done) {
-      User.find({order: 'seq', where: {name: {'gte': null},
-      }}, function(err, users) {
-        should.not.exist(err);
-        users.should.have.property('length', 0);
-        done();
+    bdd.itIf(!isCassandraConnector, 'should support string "gte" that is satisfied by null value',
+      function(done) {
+        User.find({order: 'seq', where: {name: {'gte': null}}}, function(err, users) {
+          should.not.exist(err);
+          users.should.have.property('length', 0);
+          done();
+        });
       });
-    });
 
     it('should support string "gte" that is satisfied', function(done) {
-      User.find({order: 'seq', where: {name: {'gte': 'Paul McCartney'},
-      }}, function(err, users) {
+      User.find({order: 'seq', where: {name: {'gte': 'Paul McCartney'}}}, function(err, users) {
         should.not.exist(err);
         users.should.have.property('length', 4);
-        users[0].name.should.equal('Paul McCartney');
+        for (var ix = 0; ix < users.length; ix++) {
+          (users[ix].name >= 'Paul McCartney').should.be.true();
```

Comment:
greaterThanOrEqual does not work for strings. will see if there is something else.